src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class Report {
```

Comment:
The report is one of the main entities in our project why not make it a model?

Reply:
When we will store reports in db we will need it for sure

Reply:
for now this will be overcomplication

you can leave the `Report` model if you insist
but it should have 1 field `String content`

then you will pass this content to writer